frontend/src/views/organisation/workflows/MoOrganisationUnitCreate.vue

```diff
@@ -115,14 +115,14 @@ export default {
         let vm = this
         this.isLoading = true
 
-        this.addresses.push(this.postAddress, this.phone)
-        this.addresses.forEach(a => {
+        let details = [this.postAddress, this.phone, ...this.addresses]
+        details.forEach(a => {
           if (!a.validity) {
```

Comment:
We potentially still have the issue here, don't we? One option is to use `map` instead of `forEach`; by allocating new objects, you won't leave polluted state behind.